src/scenes/titlescreen.ts

```diff
@@ -1,37 +1,45 @@
-import { Nullable, UUID } from "definitions/utils";
+import { float, Nullable } from "definitions/utils";
 import SessionManager from "managers/SessionManager";
 import Phaser from "phaser";
 import { interactify } from "../utilities/utils";
 import { ScreenHeight, ScreenWidth } from "../core/main";
+import Image = Phaser.GameObjects.Image;
 
 export class TitleScreen extends Phaser.Scene {
-    private title: Nullable<Phaser.GameObjects.Text>;
     private button: Nullable<Phaser.GameObjects.Image>;
 
     public constructor() {
         super("TitleScreen");
-        this.title = null;
         this.button = null;
     }
 
     public preload(): void {
         this.load.image("createLobby", "assets/create.png");
+        this.load.image("titlescreen", "assets/sabo.png");
     }
 
     public create(): void {
-        this.title = this.add
-            .text(ScreenWidth / 2, ScreenHeight / 4, "Saboteur", {
-                font: "Arial Black",
-                fontSize: 38,
-                color: "#ffffff",
-                stroke: "#000000",
-                strokeThickness: 8,
-                align: "center",
-            })
-            .setOrigin(0.5);
-        this.button = this.add.image(
+        this.add
+            .rectangle(0, 0, ScreenWidth, ScreenHeight, 0x000000)
+            .setOrigin(0);
```

Comment:
you can also set the background color of the entire phaser game black in the main.ts, then you can remove this fullscreen rectangle.